doc/configuration/TLS-hardening.md

```diff
@@ -44,8 +44,8 @@ In order to change the list, please find the following lines:
           ]}
 ```
 
-By default only TLS 1.2 is enabled, as 1.3 is not supported by OTPs older than 22.0.
-If you are using OTP 22.0 or newer, you may remove leading `%` before `'tlsv1.3'`.
+By default only TLS 1.2 is enabled, as 1.3 is not supported by OTPs older than 23.0.
+If you are using OTP 23.0 or newer, you may remove leading `%` before `'tlsv1.3'`.
```

Comment:
I think we could remove this information and just remove the comment in `rel/files/app.config`, because only OTP 23 and higher are supported now (also editing out the comment above).

Reply:
Done. Thanks.

Reply:
Discussed. Updated.